src/test/java/NatashaTest.java

```diff
@@ -0,0 +1,80 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class NatashaTest {
+    private WebDriver driver;
+    private final String SITE_NAME = "AskOmDch – Become a Selenium automation expert!";
+
+
+    @BeforeMethod
+    public void beforeMethod() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://askomdch.com/");
+    }
+
+    @AfterMethod
+    public void afterMethod() {
+        driver.quit();
+    }
+
+
+    @Test
+    public void VerifySaleIcon_TC_001() throws InterruptedException {
+        Assert.assertEquals(driver.getTitle(), SITE_NAME);
+        driver.findElement(By.xpath("//span[@class='onsale']")).isDisplayed();
+        Thread.sleep(2000);
+    }
```

Comment:
we don't need "sleep" at the end of each method because the test passed and no more codes.